test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -154,17 +166,25 @@ async function setup(t, locale, fileName) {
   
   await renderWidget({
     ...options,
-    'language': locale
+    'language': locale,
+    assertNoEnglishLeaks: false,
   });
+
+  await t.expect(Selector('#okta-sign-in').visible).eql(true);
 }
 
 const testEnglishLeaks = (mockIdxResponse, fileName, locale) => {
   test.requestHooks(mockIdxResponse)(`${fileName} should not have english leaks`, async t => {
+    const hasContent = !mocksWithoutContent.includes(fileName);
     await setup(t, locale, fileName);
-    const viewTextExists = await Selector('#okta-sign-in').exists;
+    if (!hasContent) {
+      return;
+    }
+    await t.expect(Selector('form').exists).eql(true);
     //Use innerText to avoid including hidden elements
-    let viewText = viewTextExists && await Selector('#okta-sign-in').innerText;
-    viewText = viewText && viewText.split('\n').join(' ');
+    let viewText = await Selector('#okta-sign-in').innerText;
+    await t.expect(viewText.length).gt(0, 'Inner text of #okta-sign-in should not be empty');
```

Comment:
`viewText` should not be empty, otherwise we can have false positive test results